Point_set_processing_3/examples/Point_set_processing_3/lfs_example_tuple.cpp

```diff
@@ -0,0 +1,55 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/estimate_lfs.h>
+#include <CGAL/IO/read_points.h>
+
+#include <vector>
+#include <utility> // defines std::pair
+#include <tuple>
+
+// types
+typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
+typedef Kernel::FT FT;
+typedef Kernel::Point_3 Point;
+typedef Kernel::Vector_3 Vector;
+
+// Point with normal vector stored in a std::pair.
+typedef std::tuple<Point, Vector, FT> Point_with_normal_and_lfs;
+
+// Concurrency
+typedef CGAL::Parallel_if_available_tag Concurrency_tag;
+
+
+int main(void)
+{
+
+  // read xyz
+  const std::string filename = "/home/daozi/CGAL/LFS-example/frog.xyz";
```

Comment:
please fix that.  See how other examples do it using ` CGAL::data_file_path()` that takes a file from `cgal/Data/data/`